worker/src/handles/TcpConnectionHandle.cpp

```diff
@@ -232,6 +233,26 @@ void TcpConnectionHandle::Write(
 		return;
 	}
 
+#ifdef MS_LIBURING_ENABLED
+	{
+		if (!DepLibUring::liburing->IsActive())
+		{
+			goto write_libuv;
+		}
+
+		auto prepared = DepLibUring::liburing->PrepareWrite(this->fd, data1, len1, data2, len2, cb);
```

Comment:
Could you add some long comment here explaining why this could fail?

Reply:
io_uring has a fixed capacity which is registered upon initialization. If the rings are in use completely at the moment of requesting, then we just fallback to libuv for this packet.

This should not happen as we are creating 1024  * 4 entries, meaning we should be able to send up to 1024 * 4 RTP packets in one shot. Usually a single Router has less than so many Consumers so this should not be a problem.